src/main/java/org/folio/marc4ld/configuration/property/Marc4BibframeRules.java

```diff
@@ -6,18 +6,27 @@
 import java.util.Map;
 import java.util.Set;
 import lombok.Data;
+import lombok.NoArgsConstructor;
 import org.springframework.boot.context.properties.ConfigurationProperties;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.context.annotation.PropertySource;
+import org.springframework.context.annotation.PropertySources;
 
 @Data
 @Configuration
 @ConfigurationProperties
-@PropertySource(value = "classpath:marc4bibframe.yml", factory = YamlPropertySourceFactory.class)
+@PropertySources({
+  @PropertySource(value = "classpath:marc4bibframe.yml", factory = YamlPropertySourceFactory.class),
+  @PropertySource(value = "classpath:marc4bibframeAutority.yml", factory = YamlPropertySourceFactory.class),
+  @PropertySource(value = "classpath:label.yml", factory = YamlPropertySourceFactory.class)
+})
 public class Marc4BibframeRules {
 
   private Map<String, List<FieldRule>> fieldRules;
```

Comment:
I think we can rename this variable to `bibFieldRules` and `bibSharedRules`, to make the distinction between Bib rules and Authority rules.

Similarly, the directories and packages can be created for "bib" and "authority". 
That is going to be a lot of changes (moving files from one directory to another). So, we can do it separately as a "cleanup" ticket after merging this PR.